### ISSUE TITLE
All popups full height on mobile

### ISSUE THREAD

**Issue by CZonin** [thomasloven/hass-browser_mod#402]
My Home Assistant version: 2022.8.7

**What I am doing:**
When using a popup card on my dashboard where the content doesn't take up the full height of the screen, the popup still extends to fullscreen.

**What I expected to happen:**
The popup background will wrap the content height and be centered vertically on the screen.

**What happened instead:**
The popup background takes up the whole screen

**Minimal** steps to reproduce:

1.  Add popup card to dashboard
2. Add a few entities to the popup card
3. Open the popup card

```
type: custom:popup-card
entity: sensor.blinds_battery_average
card:
  type: vertical-stack
  cards:
    - type: custom:mushroom-entity-card
      entity: sensor.dining_area_blinds_battery
      tap_action:
        action: more-info
      hold_action:
        action: none
      double_tap_action:
        action: none
      name: Dining Area
      icon: mdi:battery
      icon_color: teal
    - type: custom:mushroom-entity-card
      entity: sensor.kitchen_blinds_battery
      name: Kitchen
      icon: mdi:battery
      icon_color: teal
    - type: custom:mushroom-entity-card
      entity: sensor.kitchen_door_blinds_battery
      name: Kitchen Door
      icon: mdi:battery
      tap_action:
        action: more-info
      hold_action:
        action: none
      double_tap_action:
        action: none
      icon_color: teal
    - type: entities
      entities:
        - type: divider
    - type: custom:mushroom-entity-card
      entity: sensor.kitchen_door_blinds_light_level
      icon: mdi:sun-wireless
      name: Kitchen Door - Solar
      tap_action:
        action: more-info
      hold_action:
        action: none
      double_tap_action:
        action: none
      icon_color: teal
dismissable: true
title: Blinds Battery
size: normal
```

![Screenshot_20220906_095410_Home Assistant](https://user-images.githubusercontent.com/67965669/188654652-69678b7f-6c31-4fdb-8cf6-04450967f10a.jpg)

---

**By replacing the space in the checkboxes ([ ]) with an X below, I indicate that I:**

- [x] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [x] Have made sure I am using the latest version of the plugin.

- [x] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [x] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by thomasloven** [comment]
Should be fixed in 2.1.2.

**Comment by CZonin** [comment]
All fixed, thanks @thomasloven
